packages/webview_flutter/webview_flutter_web/example/integration_test/webview_flutter_test.dart

```diff
@@ -5,6 +5,10 @@
 import 'dart:html' as html;
 import 'dart:io';
 
+// FIX (dit): Remove these integration tests, or make them run. They currently never fail.
```

Comment:
Ah, I didn't recognize this when I updated these tests. Thanks for noticing this!